e2e/cypress/integration/ui/settings/04_project_mgmt.spec.ts

```diff
@@ -0,0 +1,206 @@
+interface Project {
+  id: string;
+  name: string;
+  type: string;
+}
+
+// assume 2.0 if not in CI. if you wish something different start cypress with
+// IAM_VERSION set to what you are testing.
+if (Cypress.env('IAM_VERSION') === undefined) {
+  Cypress.env('IAM_VERSION', 'v2.0');
+}
+
+const describeIAMV2P1 = Cypress.env('IAM_VERSION') === 'v2.1' ? describe : describe.skip;
+
+describeIAMV2P1('project management', () => {
+  let adminToken = '';
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'cypress-test';
+  const projectID = `${cypressPrefix}-project1-${now}`;
+  const projectName = `${cypressPrefix} project1 ${now}`;
+  const ruleID = `${cypressPrefix}-rule-${now}`;
+  const ruleName = `${cypressPrefix} rule ${now}`;
+
+  before(() => {
+    cy.adminLogin('/settings/projects').then(() => {
+      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+      adminToken = admin.id_token;
+      cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
+    });
+  });
+
+  beforeEach(() => {
+    cy.restoreStorage();
+  });
+
+  afterEach(() => {
+    cy.saveStorage();
+  });
+
+
+  it('list projects', () => {
+    cy.request({
+      auth: { bearer: adminToken },
+      method: 'GET',
+      url: '/apis/iam/v2beta/projects'
+    }).then((response) => {
+      expect(response.status).to.equal(200);
+      response.body.projects.map((project: Project) => project.id)
+        .forEach((id: string) => {
+        cy.get('chef-table chef-td').contains(id);
+      });
+    });
+  });
+
+  it('can create a project', () => {
+    cy.get('#create-button').contains('Create Project').click();
+    cy.get('app-create-object-modal chef-modal').should('have.class', 'visible');
+
+    // we increase the default delay to mimic the average human's typing speed
+    // only need this for input values upon which later test assertions depend
+    // ref: https://github.com/cypress-io/cypress/issues/534
```

Comment:
So where do we actually need the typing helper? Are we just doing it in places its failed without it before? If we need it in a bunch of places, we should make a helper.

Reply:
It's for any input that must be entered correctly for later assertions to pass. So if I create a user named `test` but it gets typed `tst`, when the test tries to click on the name link, it'll be expecting `test`, not `tst`.

Helper seems like a good idea for consistency

Reply:
Might make that a follow-up though because I'm a bit Cypress'd out 😅